index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
This change is not scoped to just Bunsen as the title of the PR indicates it would/should be, nor are you able to do such scoping from within this repo. If such a change is desired for Bunsen then this change should be made in the Bunsen repo and not here.